wiki/Getting-Started-UIKit.md

```diff
@@ -0,0 +1,248 @@
+# Interested in seeing examples of Workflow in action?
+Start by cloning the repo and checking out the 'WorkflowExample' scheme. This should give you a decent idea of how the library works.  If you want to create a new project, read on.
+
+# CocoaPods with Programmatic Views
+
+## Adding the dependency
+```ruby
+pod 'DynamicWorkflow/UIKit'
+```
+Add the above line to your Podfile.
+
+For more installation instructions, see the wiki on [installation](https://github.com/wwt/Workflow/wiki/Installation).
+
+## Create your view controllers
+Create two view controllers that inherit from [UIWorkflowItem<I, O>](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Classes/UIWorkflowItem.html).
+
+```swift
+import UIKit
+import Workflow
+import WorkflowUIKit
+
+class FirstViewController: UIWorkflowItem<String, String>, FlowRepresentable {
+    private let name: String
+    private let emailTextField = UITextField()
+    private let welcomeLabel = UILabel()
+    private let saveButton = UIButton()
+
+    required init(with name: String) {
+        self.name = name
+        super.init(nibName: nil, bundle: nil)
+
+        view.backgroundColor = .systemGray5
+
+        welcomeLabel.text = "Welcome \(name)!"
+
+        emailTextField.backgroundColor = .systemGray3
+        emailTextField.borderStyle = .roundedRect
+        emailTextField.placeholder = "Enter email..."
+
+        saveButton.setTitle("Save", for: .normal)
+        saveButton.setTitleColor(.systemBlue, for: .normal)
+        saveButton.addTarget(self, action: #selector(savePressed), for: .touchUpInside)
+
+        view.addSubview(welcomeLabel)
+        view.addSubview(emailTextField)
+        view.addSubview(saveButton)
+
+        welcomeLabel.translatesAutoresizingMaskIntoConstraints = false
+        welcomeLabel.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+        welcomeLabel.topAnchor.constraint(equalTo: view.topAnchor, constant: 100).isActive = true
+
+        emailTextField.translatesAutoresizingMaskIntoConstraints = false
+        emailTextField.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+        emailTextField.topAnchor.constraint(equalTo: welcomeLabel.bottomAnchor, constant: 16).isActive = true
+        emailTextField.widthAnchor.constraint(equalToConstant: 300).isActive = true
+
+        saveButton.translatesAutoresizingMaskIntoConstraints = false
+        saveButton.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+        saveButton.topAnchor.constraint(equalTo: emailTextField.bottomAnchor, constant: 24).isActive = true
+    }
+
+    required init?(coder: NSCoder) { nil }
+
+    @objc
```

Comment:
This violates our styleguide. I weirdly cannot find a link to the exact rule but annotations go on the same line as the function declaration, they aren't broken apart above it. You can see several examples if you search the styleguide for `@`. This fits with SwiftUI conventions like how you have
```swift
// RIGHT
@State var thing: String = ""

// WRONG
@State
var thing: String = ""
```

Reply:
I could not find a specific rule for this either, but I was ultimately able to find it played out in some of the examples for other rules. Based on this, I think we should consider adding this as a rule in the styleguide so that it is more explicit.

Reply:
Issue tracked with: #42